translations/xblock-drag-and-drop-v2/drag_and_drop_v2/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -1,18 +1,15 @@
-# SOME DESCRIPTIVE TITLE.
```

Comment:
Verify that cleaning is working after the fix of https://github.com/openedx/i18n-tools/pull/136